### DISCUSSION TITLE
Making some accounts imap only (disabling smtp)

### DISCUSSION THREAD

**Discussion by doubleaxe** [docker-mailserver/docker-mailserver#4299] [Q&A] [answered]
Hello.
I have some accounts like archive@domain.com or test@domain.com, which are used to store archive/test emails and won't be used for sending. I'm looking for a way to make these accounts imap only, thus I will not have to worry about strong passwords for these accounts.

**Comment by polarathene** [comment, marked as answer]
You can [generate passwords that are secure](https://getapassphrase.com/generate/). For example `tutoring zombie benefits stormy hawk` has 48 bits of entropy. Your attack vector is remote login which is slow, couple that with Fail2Ban and no one is realistically going to brute force you with such a slow attack.

Postfix (SMTP) relies on Dovecot (IMAP) for account authentication, we do not have a way that I'm aware of to configure what you're asking for.
